Error when updating

Using tz_data 0.5.15.

This night I got a couple of error log messages from two different applications deployed on the same host. There was around three hours between the errors and I haven't noticed any other issues with the apps or host.

One of the messages:

```
[ERROR] <0.2341.0>: GenServer :tzdata_release_updater terminating 
** (File.Error) could not stream "/path/to/app/lib/tzdata-0.5.15/priv/tmp_downloads/339626_44358462//pacificnew": no such file or directory     
(elixir) lib/file/stream.ex:79: anonymous fn/2 in Enumerable.File.Stream.reduce/3     
(elixir) lib/stream.ex:1270: anonymous fn/5 in Stream.resource/3     
(elixir) lib/stream.ex:1433: Enumerable.Stream.do_each/4     (elixir) lib/enum.ex:1841: Enum.reverse/1     
(elixir) lib/enum.ex:2599: Enum.to_list/1     (tzdata) lib/tzdata/parser.ex:16: Tzdata.Parser.process_file/1     
(tzdata) lib/tzdata/basic_data_map.ex:8: anonymous fn/2 in Tzdata.BasicDataMap.from_files_in_dir/1     
(elixir) lib/enum.ex:1273: Enum."-map/2-lists^map/1-0-"/2     (elixir) lib/enum.ex:1273: Enum."-map/2-lists^map/1-0-"/2     
(tzdata) lib/tzdata/basic_data_map.ex:8: Tzdata.BasicDataMap.from_files_in_dir/1     
(tzdata) lib/tzdata/data_builder.ex:32: Tzdata.DataBuilder.do_load_and_save_table/4     
(tzdata) lib/tzdata/release_updater.ex:46: Tzdata.ReleaseUpdater.poll_for_update/0 Last message: :check_if_time_to_update 
```

The other is similar and can also not find the `pacificnew` file.

The only other tzdata logs I can find is from around 1 second before the error saying

```
Tzdata.ReleaseUpdater  [INFO]  <0.2341.0>: tzdata release in place is from a file last modified Mon, 23 Oct 2017 15:19:20 GMT. Release file on server was last modified Tue, 16 Jan 2018 17:43:58 GMT. 
```

In the latest tzdata from https://www.iana.org/time-zones, the `pacificnew` is not included, so perhaps it should just be removed from [BasicDataMap](https://github.com/lau/tzdata/blob/master/lib/tzdata/basic_data_map.ex#L6)?

--- a/lib/tzdata/basic_data_map.ex
+++ b/lib/tzdata/basic_data_map.ex
@@ -3,7 +3,7 @@ defmodule Tzdata.BasicDataMap do
 
   alias Tzdata.Parser
   alias Tzdata.ParserOrganizer, as: Organizer
-  @file_names ~w(africa antarctica asia australasia backward etcetera europe northamerica pacificnew southamerica)s
+  @file_names ~w(africa antarctica asia australasia backward etcetera europe northamerica southamerica)s
   def from_files_in_dir(dir_name) do
     Enum.map(@file_names, fn file_name -> {String.to_atom(file_name), Parser.read_file(file_name, dir_name)} end)
     |> make_map

